SmartDeviceLink/public/SDLLifecycleConfiguration.m

```diff
@@ -72,13 +64,13 @@ - (instancetype)initDefaultConfigurationWithAppName:(NSString *)appName fullAppI
     _allowedSecondaryTransports = SDLSecondaryTransportsTCP;
 
     _fullAppId = fullAppId;
-    _appId = fullAppId != nil ? [self.class sdl_shortAppIdFromFullAppId:fullAppId] : appId;
+    _appId = [self.class sdl_shortAppIdFromFullAppId:fullAppId];
 
     return self;
 }
 
-- (instancetype)initDefaultConfigurationWithAppName:(NSString *)appName fullAppId:(nullable NSString *)fullAppId appId:(nullable NSString *)appId ipAddress:(NSString *)ipAddress port:(UInt16)port {
-    SDLLifecycleConfiguration *config = [self initDefaultConfigurationWithAppName:appName fullAppId:fullAppId appId:appId];
+- (instancetype)initDefaultConfigurationWithAppName:(NSString *)appName fullAppId:(NSString *)fullAppId ipAddress:(NSString *)ipAddress port:(UInt16)port {
```

Comment:
Shouldn't this be `initDebugConfiguration...`?

Reply:
I just removed this method as it was only being called by one other method